README.rst

```diff
@@ -161,6 +161,43 @@ fault by adding the ``-v`` (verbose) flag::
 
 I need to rebuild this wheel to link with dual-architecture libraries.
 
+Making dual-architecture binaries
+=================================
+
+Modern Mac wheels can be either ``arm64e`` (M1 ARM), ``x86_64`` (64-bit Intel) or
+both ("universal").
+
+Building an entire Python wheel as dual-architecture can be difficult, perhaps
+because you need to link different libraries in the two cases, or you need
+different compiler flags.
```

Comment:
maybe add "or because you build for `arm64` on native (M1) CI". May or may not want to mention Cirrus CI, which isn't very well-known yet but offers free M1 CI which works very well.

Reply:
Done.